bokehjs/src/lib/models/widgets/abstract_slider.ts

```diff
@@ -153,6 +159,7 @@ abstract class AbstractBaseSliderView extends OrientedControlView {
 
     this.group_el = div({class: inputs.input_group}, this.title_el, this.slider_el)
     this.el.appendChild(this.group_el)
+    this._has_finished = true
```

Comment:
Why is this necessary?

Reply:
Because we set it to false if mathjax is still loading after first layout
https://github.com/bokeh/bokeh/blob/40ca075d40f951ec604d61be8ba43ddffbc17c5e/bokehjs/src/lib/models/widgets/widget.ts#L55